content/chainguard/chainguard-enforce/vulnerabilities/vulns-in-enforce/index.md

```diff
@@ -0,0 +1,63 @@
+
+## Chainguard Enforce automatically ingests Vulnerability reports out of the box
+
+When you run a workload with a supported Container Runtime (including EKS, GKE, and Cloud Run) monitored by Chainguard Enforce, it will first check if there are already vulnerability reports attached to the workload’s container images. We support signed cosign vulnerability reports (e.g. a vulnerability report uploaded using `cosign attest` as an in-toto attestation). Nowadays Chainguard Enforce supports Grype’s custom scan report format in JSON format. The result of a Grype scanning operation is included within the cosign vulnerability attestation specification. That attestation is supposed to be signed and include additional metadata about the scanner used to generate the vulnerability report. Enforce also checks the timestamp of the vulnerability attestation ignoring all reports whose age is older than 24 hours. If all these conditions are satisfied, Enforce will ingest that scan report.
+
+
+## Vulnerability report Generation in Chainguard Enforce
+
+If there are container images without an existing vulnerability report, Enforce will assist by automatically creating a vulnerability report using [Grype](https://github.com/anchore/grype). This means that you don't have to worry about generating the scanning reports yourself or performing any additional steps. Enforce takes care of it for you, ensuring that you have comprehensive information about the vulnerabilities discovered for each image. To create a vulnerability report, Enforce relies on a previously generated or ingested SBOM for each image. By doing so, Enforce scanning focuses on the list of available packages used in your workloads.
+
+Generated vulnerability reports will be clearly indicated in the Enforce user interface. In the Vulnerabilities section of the Console, you will find a list of all the discovered vulnerabilities of the packages running on every cluster overseen by Enforce. The list of vulnerabilities will be listed by CVE ID indicating its severity level and information about the package where the vulnerability has been detected. Another column `Images` indicates the amount of images affected by a vulnerability on your clusters.
+
+
+![Screenshot of a portion of the Vulnerabilities overview page in the Chainguard Enforce console, showing a list of the discovered vulnerabilities.](list-vulns.png)
+
+To streamline this process, we don’t require any user action to enable Vulnerability report generation. As soon as it detects a container image running in one of your clusters without a Vulnerability report, Chainguard Enforce will generate one for you.
```

Comment:
```suggestion
To streamline this process, Chainguard Enforce does not require any user action or configuration to generate Vulnerability reports. As soon as Enforce detects a container image running in one of your clusters without a Vulnerability report it will generate one for you.
```